### ISSUE TITLE
Should we add `aria-hidden="true"` to the page margin boxes?

### ISSUE THREAD

**Issue by MurakamiShinyu** [vivliostyle/vivliostyle.js#1561]
This issue is related to the accessibility of the PDF output. By adding `aria-hidden="true"` to the page margin boxes, we can prevent screen readers from announcing the header and footer content, which may be redundant or confusing for users. However, we also need to consider cases where this content should be announced, so further discussion is needed.

From https://github.com/vivliostyle/vivliostyle.js/issues/1555#issuecomment-3299653057, thanks @danvim:

> Here are some resources I found that talks about marking headers and footers as `Artifact`s (probably corresponding to `aria-hidden`):
> 
> -   <https://www.w3.org/TR/WCAG20-TECHS/PDF14.html>
> -   <https://www.davidmacd.com/blog/pdf-headers-footers.html>
